lib/tessel/tessel.js

```diff
@@ -95,15 +95,13 @@ Tessel.prototype.receive = function(remote) {
 
 Tessel.prototype.simpleExec = function(command) {
   var self = this;
-  return new Promise(function(resolve, reject) {
-    // Stop processes and delete everything in the folder
-    return self.connection.exec(command)
-      .then(function(remoteProcess) {
-        return self.receive(remoteProcess).then(function(received) {
-          return resolve(received.toString());
-        }).catch(reject);
+  // Stop processes and delete everything in the folder
+  return self.connection.exec(command)
```

Comment:
Just cleaning up a little bit - there is no need to wrap in the Promise constructor.